### PR TITLE
provided 3 versions of a pangram checker, could use a 4th

### PR DESCRIPTION
**Pull request by alison-morgan**

### CODE REVIEW

File: whiteBoarding/pangram/eawPangram.js [hunk 1]
```diff
@@ -1 +1,40 @@
 //  this is an example file to build out for the pangram checker
+`this is an example of a pangram checker if you know there won't be any
+repeated characters or non-alpha characters`.
+
+`function isPangram(str) {
```
Comment (ed42311) [inline whiteBoarding/pangram/eawPangram.js]: are there backticks around all of these functions?

File: whiteBoarding/fizzbuzz/fizzbuzz.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+`1st) return fizzbuzz. 2nd) return fizzbuzz but in lieu of fizz,buzz,fizzbuzz return key values from this provided object => let sounds = {
```
Comment (ed42311) [inline whiteBoarding/fizzbuzz/fizzbuzz.md]: in the other files the instructions for the exercises are in a file called `instructions.md`.  Also these files have an outline of what needs to be done and a title and links to attempts.  Structure within files is often there for a reason.

File: whiteBoarding/fizzbuzz/fizzbuzz.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+                             cat:'meow', fizz
+                             dog:'woof', buzz
```
Comment (ed42311) [inline whiteBoarding/fizzbuzz/fizzbuzz.md]: this formatting needs a bit of work, build it into a code block in your markdown or at least bracket it close to the edge of the page.

File: whiteBoarding/palindrome/palindrome.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+`return a boolean value for whether or not a given string is a
```
Comment (ed42311) [inline whiteBoarding/palindrome/palindrome.md]: is this meant to be the instructions file?  see above comments ^

**Comment by alison-morgan** [comment]
@ed42311 I made some changes, check em!

**Pull request merged**